avocado/core/loader.py

```diff
@@ -63,6 +63,33 @@ class MissingTest(object):
     """
 
 
+def parse_filter_by_tags(filter_by_tags):
+    """
+    Parses the various filter by tags in "command line" format
+
+    The filtering of tests usually happens my means of "--filter-by-tags"
+    command line options, and many can be given.  This parses the contents
+    of those into a list of must/must_not pairs, which can be used directly
+    for comparisons when filtering.
+
+    :param filter_by_tags: params in the format given to "-t/--filter-by-tags"
+    :type filter_by_tags: list of str
```

Comment:
I think the parameter here cannot be a string. If it's a string the iteration loop (line 80) will iterate over each character.

Reply:
Yep. I misread it. Should I merge it?

Reply:
If that's the only concern, yes!